### ISSUE TITLE
question for the evaluation metric

### ISSUE THREAD

**Issue by liuxindazz** [liarba/scene_recognition#2]
I'm confused about the mean accuracy over categories. It's the means of diagonal values of the confusion matrices in a paper you referenced.
That means Precision = TP/(TP+FP) for every category, and then take the average value of them？
I can't reproduce the result even though the baseline (RGB Alexnet 42.6%). The means of diagonal values of the confusion matrices is 38% when I finetune the Alexnet with SUN RGB images and get 42% accuracy. (accuracy = predicted right number/total number)
Which metric is used in your paper, the means of diagonal values or the accuracy?

**Comment by liarba** [comment]
We use the mean of diagonal values of confusion matrix in evaluation.
Something I think might be useful: the high baseline is partly due to that we use places CNN as our pretrained model. If we use ImageNet as our pretrained model, the mean accuracy is about 40%. (We didn't list in paper but we actually also did all experiments with ImageNet pretrained model, the same conclusion can be drawed. However the final mean accuracy will be about 2% less.)

**Comment by liuxindazz** [comment]
Thanks for your reply, I also tried the Places CNN but failed.
Could you provide the baseline caffemodel and deploy.prototxt?

**Comment by liuxindazz** [comment]
I tried the Places CNN again for SUN RGB images and got 41.6% average precision.
Maybe I could go on tuning the model to achieve the baseline.
Do you have some effective strategies to improve performance?

**Comment by liuxindazz** [comment]
I have already achieved the RGB finetune baseline using the Places CNN.
Thank you!!
